server/test/users.test.js

```diff
@@ -36,3 +37,33 @@ describe('GET /users/request', () => {
     });
   });
 });
+
+describe('GET /users/request/requestId', () => {
+  before(() => {
+    requests.push(...testRequests);
+  });
+
+  it('should return a request if a valid id is sent', (done) => {
+    request(app)
+      .get('/api/v1/users/requests/2')
+      .expect(200)
+      .expect((res) => {
+        expect(res.body.status).to.equal('success');
+        expect(res.body.message).to.equal('Your request has been found');
+        expect(res.body.request).to.be.an('object');
+      })
+      .end(done);
+  });
+
+  it('should return an error message if an invalid id is sent', (done) => {
+    const id = 22;
+    request(app)
+      .get(`/api/v1/users/requests/${id}`)
+      .expect(404)
+      .expect((res) => {
```

Comment:
What's this asserting?

Reply:
It is an assertion to check that the right error  status code with its appropriate message is being sent

Reply:
Okay. This flies